components/ImaSdkTask.brs

```diff
@@ -43,19 +43,45 @@ sub runLoop()
         m.top.video.ObserveField(field, m.port)
     end for
 
+    ' track last video position to support snapback
+    m.lastVideoTime = m.top.video.position
+
     while true
         msg = Wait(1000, m.port)
         ' continue running task until video is no longer available
         if m.top.video = invalid then exit while
 
         m.streamManager.onMessage(msg)
 
+        currentContentTime = m.streamManager.getContentTime(m.top.video.position * 1000) / 1000
```

Comment:
Is this var used anywhere?

Reply:
Nope, good catch. This is an artifact from copying the example snapback, I'll remove it.